### PR TITLE
Add tests, linter and doc

### PR DESCRIPTION
**Pull request by bySabi**
@tmcw-up-for-adoption please review this PR.

I add ...
- linter setup
- initial test
- some Doc tweaks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+  "version": "2.1.0",
+  "description": "Karma Adapter to tape testing framework",
   "main": "index.js",
   "scripts": {
     "clean": "rimraf lib",
     "build:parser": "browserify src/parser.js --s tapParser -o lib/parser.js",
     "build-npm": "npm run clean && mkdirp lib && npm run build:parser",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "karma": "karma start",
+    "lint": "eslint . --ext .js,.jsx",
```
Comment (tmcw) [inline package.json]: This project doesn't include any es6 or jsx code, right? This line and the choice of eslint-standard-deviation seems off in that respect; including es6 code would pass lint but fail when it actually runs

Comment (bySabi) [inline package.json]: You are right, sorry! ... fixed on latest commit

**Comment by bySabi** [comment]
@tmcw-up-for-adoption  I update the PR

Thanks

**Comment by tmcw** [comment]
👍 Thanks, merging!

**Pull request merged**